### ISSUE TITLE
Tamed dilophosaurus just want you all to themselves?

### ISSUE THREAD

**Issue by Canus8** [JurassiCraftTeam/JurassiCraft2#563]
both times I have made dilos, the attack me.  They are tamed to me, and are supposed to protect you from attackers, but they seem to just want to attack me.  I am not sure if this is a bug or not, but it seems a little counter-intuitive for them to be able to defend you if they are only going to attack you themselves.  I have not gotten to test this for Velociraptors, as all of mine keep dying in the low security fence base of their enclosure before they were even 5 days old.

I also could note that I am mostly using peaceful, so I don't know if that is a factor.

The incident happened with hunting only when hungry set to true, and when set to false, and in BOTH situations, it happens even when the dilophosaurs aren't hungry.

**Comment by PT400C** [comment]
@Canus8 afaik that's their normal behaviour and personality. Some of them are agressive and other's aren't. If you would hatch some more some should also be fairly friendly.

**Comment by Canus8** [comment]
Of course, that is if I could keep them alive!  my automated feeder keeps breaking, and I think my last pair starved.  The first pair I killed because they attacked, and glitched into the ground and the chicken farm that supplies the automated feeder system.  Kind of unfortunate.  I was hoping to keep at least one Jurassicraft dinosaur in my world.  raptors die when they brush against the low security fence base, brachiosaurus lag out the world when they reach juvenile age, triceratops and parasaurolophus just vanish, without any explanation (they had sufficient water and food and no predators), mussaurus vanish (although this might be related to the Velociraptor issue), and T rex are just too big for what I have right now.  And gallimimus run of to who knows where, and then they vanish too.  that is, unless they spawn naturally, in which case they all lag out the world.

**Comment by Gamebrothers4** [comment]
Maybe use a Jc feeder then :?
